src/data/support-title.csv

```diff
@@ -1,65 +1,65 @@
 "text", "translation"
-"[トレセン学園]",""
-"[やれやれ、お帰り]",""
-"[努力は裏切らない！]",""
-"[テッペンに立て！]",""
-"[副会長の一刺し]",""
-"[デジタル充電中+]",""
-"[検証、開始]",""
-"[カワイイ＋カワイイは～？]",""
-"[雨の独奏、私の独創]",""
-"[鍛えぬくトモ]",""
-"[鍛えて、応えて！]",""
-"[シチーガール入門♯]",""
-"[生体Aに関する実験的研究]",""
-"[0500・定刻通り]",""
-"[波立つキモチ]",""
-"[マーベラス☆大作戦]",""
-"[運の行方]",""
-"[幸せと背中合わせ]",""
-"[目線は気にせず]",""
-"[…ただの水滴ですって]",""
-"[一流プランニング]",""
-"[共に同じ道を！]",""
-"[見習い魔女と長い夜]",""
-"[パリピ・ぱーりないと！]",""
-"[準備運動は怠るべからず]",""
-"[むじゃむじゃむじゃき]",""
-"[あなたにささげる]",""
-"[おすすめ本、あります！]",""
-"[世界の真珠、その名は]",""
-"[日本一のステージを]",""
-"[輝く景色の、その先に]",""
-"[夢は掲げるものなのだっ！]",""
-"[不沈艦の進撃]",""
-"[ロード・オブ・ウオッカ]",""
-"[千紫万紅にまぎれぬ一凛]",""
-"[パッションチャンピオーナ！]",""
-"[待望の大謀]",""
-"[天をも切り裂くイナズマ娘！]",""
-"[感謝は指先まで込めて]",""
-"[飛び出せ、キラメケ]",""
-"[B・N・Winner!!]",""
-"[7センチの先へ]",""
-"[Run(my)way]",""
-"[はやい！うまい！はやい！]",""
-"[一粒の安らぎ]",""
-"[これが私のウマドル道☆]",""
-"[まだ小さな蕾でも]",""
-"[うらら～な休日]",""
-"[必殺！Wキャロットパンチ！]",""
-"[ようこそ、トレセン学園へ！]",""
-"[『エース』として]",""
-"[『幸せ』が舞う時]",""
-"[『愛してもらうんだぞ』]",""
-"[夕焼けはあこがれの色]",""
-"[ターボエンジン全開宣言！]",""
-"[バカと笑え]",""
-"[迫る熱に押されて]",""
-"[その背中を越えて]",""
-"[Just keep going.]",""
-"[ふるさと直送エール！]",""
-"[押して忍べど燃ゆるもの]",""
-"[夢はホントに叶うんだ！]",""
-"[今ぞ盛りのさくら花]",""
-"[ふたつのノーブルライト]",""
+"[トレセン学園]","Tracen Academy"
+"[やれやれ、お帰り]","Oh Dear, Welcome Back"
+"[努力は裏切らない！]","Effort Does Not Betray!"
+"[テッペンに立て！]","Stand at the Top!"
+"[副会長の一刺し]","Vice President's Swift Justice"
+"[デジタル充電中+]","Digital Charging in Progress+"
+"[検証、開始]","Verification, Start"
+"[カワイイ＋カワイイは～？]","Cute + Cute is~?"
+"[雨の独奏、私の独創]","Rain's Recital, My Original"
+"[鍛えぬくトモ]","Workout Companion"
+"[鍛えて、応えて！]","Exercise and Excel!"
+"[シチーガール入門♯]","City Girl Training, Begin♯"
+"[生体Aに関する実験的研究]","Experimental Study of Lifeform A"
+"[0500・定刻通り]","0500, Just As Scheduled"
+"[波立つキモチ]","Rippling Feelings"
+"[マーベラス☆大作戦]","Marvelous ☆ Masterplan"
+"[運の行方]","Whereabouts of Luck"
+"[幸せと背中合わせ]","Back-to-Back with Happiness"
+"[目線は気にせず]","Don't Mind the Gazes"
+"[…ただの水滴ですって]","…Just a Drop of Water"
+"[一流プランニング]","First-Class Planning"
+"[共に同じ道を！]","Together on the Same Path!"
+"[見習い魔女と長い夜]","The Apprentice Witch and the Long Night"
+"[パリピ・ぱーりないと！]","Party People, Party Night!"
+"[準備運動は怠るべからず]","Thou Shan't Neglect Warming Up"
+"[むじゃむじゃむじゃき]","Disheveled Innocence"
+"[あなたにささげる]","Devoted to You"
+"[おすすめ本、あります！]","Book Recommendations Available!"
+"[世界の真珠、その名は]","Pearl of the World"
+"[日本一のステージを]","Japan's Number 1 Stage"
+"[輝く景色の、その先に]","Beyond the Brilliant Scenery"
+"[夢は掲げるものなのだっ！]","Let Your Dreams Be Known!"
+"[不沈艦の進撃]","Advance of the Unsinkable Battleship"
+"[ロード・オブ・ウオッカ]","Road of Vodka"
+"[千紫万紅にまぎれぬ一凛]","A Myriad of Colors, A Noble Bloom"
+"[パッションチャンピオーナ！]","Passion Championa!"
+"[待望の大謀]","Long-awaited Masterplan"
+"[天をも切り裂くイナズマ娘！]","Heaven-piercing Lightning Girl!"
+"[感謝は指先まで込めて]","Gratitude up to one's Fingertips"
+"[飛び出せ、キラメケ]","Jump Out and Shine"
+"[B・N・Winner!!]","B・N・Winner!!"
+"[7センチの先へ]","Beyond the 7 cm"
+"[Run(my)way]","Run (my) Way"
+"[はやい！うまい！はやい！]","Fast! Skillful! Fast!"
```

Comment:
Please look at the card art again lol

Reply:
Fair point, I did indeed forget the card art for Bakushin's here. Change Skillful for Delicious then?

Reply:
Agreed!